.github/workflows/smoke-e2e-ci-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Smoke E2E Tests
+on: [push, pull_request]
+jobs:
+  against-real-network-on-linux:
+    timeout-minutes: 30
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        node: [16, 18]
+        os: [ubuntu-latest]
+    services:
+      mina-local-network:
+        image: o1labs/mina-local-network:rampup-latest-lightnet
+        env:
+          NETWORK_TYPE: 'single-node'
+          PROOF_LEVEL: 'none'
+        ports:
+          - 3085:3085
+          - 8080:8080
+          - 8181:8181
+        # TODO: Disable logging for the container as the workaround for a long post-job-cleanup phase
+        # - Will be fixed by improving logging as part of the work on:
+        # - https://hub.docker.com/r/o1labs/mina-local-network
+        options: --log-driver=none
+    steps:
+      - uses: actions/checkout@v3
+      - name: Make helper script executable
+        run: chmod +x tests/scripts/wait-for-sync.sh
+      - name: Wait for Docker container services to be ready
+        run: ./tests/scripts/wait-for-sync.sh 8080
+      - name: Setup NodeJS ${{ matrix.node }}
+        uses: actions/setup-node@v3
```

Comment:
@mitschabaude why you think it will be faster? GitHub Workflow `jobs` are executed in parallel by default, but `job.steps` are executed sequentially, hence, in the end the order of steps doesn't really matter since we will wait for network readiness anyway.

Reply:
@mitschabaude I'll assume it is not an issue anymore.

Reply:
I somehow thought that this was waiting on something started earlier, but I guess it's started in this job and waited for, in that case of course it doesn't make a difference